contribs/github-bot/internal/requirements/reviewer_test.go

```diff
@@ -36,7 +36,7 @@ func TestReviewByUser(t *testing.T) {
 			State: github.String("APPROVED"),
 		}, {
 			User:  &github.User{Login: github.String("anotherOne")},
-			State: github.String("REQUEST_CHANGES"),
+			State: github.String("CHANGES_REQUESTED"),
```

Comment:
Looks like you forgot to use the constants you defined in utils here and in the rest of this file. ;)

Reply:
For this file, since it's a "mock test" I thought we wanted to keep the literals directly rather than using the constants... but sure, updating :)

Reply:
I think it makes sense to keep the literals directly in the test. Do you agree?

Reply:
Ok let's keep them 👍